### DISCUSSION TITLE
Debugging code

### DISCUSSION THREAD

**Discussion by anuphw** [pranjal-joshi/Screeni-py#118] [Q&A] [answered]
Hi,

I am trying to add a rsi divergence filter, but cannot know what's wrong with my code as no errors are printed when I execute the code. 

How do I disable the error suppressing? 

*Edits in classes/Screener.py*

<img width="824" alt="image" src="https://user-images.githubusercontent.com/18086525/168519140-1cc14c1e-45be-4687-a71b-b67d07e65919.png">

<img width="824" alt="image" src="https://user-images.githubusercontent.com/18086525/168519053-32ddcbaa-a069-498d-a810-ca084d48f3c4.png">

**Comment by pranjal-joshi** [comment, marked as answer]
Hi @anuphw,

I'm glad that you find the tools useful and that you are looking forward to contribute.
Try adding exception printing in the following line before an `if` statement:
https://github.com/pranjal-joshi/Screeni-py/blob/6d9f406cace8c8579fe04f7e3301de3ba7aa88b4/src/classes/ParallelProcessing.py#L239-L242
Also, You can try to debug by forcing multiprocessing to run only one child process by setting `totalConsumer=1` for easy debugging.
https://github.com/pranjal-joshi/Screeni-py/blob/6d9f406cace8c8579fe04f7e3301de3ba7aa88b4/src/screenipy.py#L257-L258
It would be great if you keep posting here about your developments for further integration. 😄
